lib/plivo/resources/applications.rb

```diff
@@ -8,6 +8,7 @@ def initialize(client, options = nil)
         @_name = 'Application'
         @_identifier_string = 'app_id'
         super
+        @_is_voice_request = true
```

Comment:
send this variable inside params map and make changes accordingly.

Reply:
Works even if it is sent by declaring at the point of initialising the class